bodo/tests/test_json.py

```diff
@@ -367,6 +366,7 @@ def read_impl(fname):
     json_write_test(test_impl, read_impl, df, "A")
 
 
+# TODO[BSE-4577]: Find and fix memory leak in to_json
 @pytest.mark.parametrize("orient", ["split", "index", "columns", "table"])
 def test_json_write_orient(test_df, orient, memory_leak_check):
```

Comment:
Did you mean to remove memory_leak_check here too?

Reply:
Yes thanks